app/pb_kits/playbook/pb_nav/_item.jsx

```diff
@@ -4,24 +4,34 @@ import PropTypes from "prop-types";
 const propTypes = {
   text: PropTypes.string.isRequired,
   link: PropTypes.string,
-  active: PropTypes.bool
+  active: PropTypes.bool,
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node
+  ])
 };
```

Comment:
We will always want to use [Flow Types](https://flow.org/) like we use in other kits instead of prop-types, so this would be:

```jsx
type Props = {
  text: String,
  link: String,
  active?: Boolean,
  children: Array<React.ReactChild> | React.ReactChild,
}
```